[2024-10-23T15:38:54.673+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag2.hello_task2 scheduled__2024-10-22T12:00:00+00:00 [queued]>
[2024-10-23T15:38:54.679+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag2.hello_task2 scheduled__2024-10-22T12:00:00+00:00 [queued]>
[2024-10-23T15:38:54.679+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-10-23T15:38:54.694+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): hello_task2> on 2024-10-22 12:00:00+00:00
[2024-10-23T15:38:54.699+0000] {standard_task_runner.py:57} INFO - Started process 408 to run task
[2024-10-23T15:38:54.702+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag2', 'hello_task2', 'scheduled__2024-10-22T12:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag2.py', '--cfg-path', '/tmp/tmpqy3s8vji']
[2024-10-23T15:38:54.703+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask hello_task2
[2024-10-23T15:38:54.757+0000] {task_command.py:410} INFO - Running <TaskInstance: my_first_dag2.hello_task2 scheduled__2024-10-22T12:00:00+00:00 [running]> on host 500df0ec66bd
[2024-10-23T15:38:55.315+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag2' AIRFLOW_CTX_TASK_ID='hello_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-10-22T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-22T12:00:00+00:00'
[2024-10-23T15:38:55.316+0000] {logging_mixin.py:150} INFO - Привет, Николай!
[2024-10-23T15:38:55.316+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-23T15:38:55.325+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_first_dag2, task_id=hello_task2, execution_date=20241022T120000, start_date=20241023T153854, end_date=20241023T153855
[2024-10-23T15:38:55.418+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-23T15:38:55.435+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
